docs-src/references/commands.md

```diff
@@ -50,27 +50,55 @@ This Command is triggered by a call to spawn a [Child Workflow Execution](/conce
 - Awaitable: Yes, a Workflow Execution can await on the action resulting from this Command.
 - Corresponding Event: [ChildWorkflowExecutionStarted](/references/events/#childworkflowexecutionstarted)
 
+:::note
+
+OSS users cannot have more than 50000 pending Child Workflows.
+Cloud users can have up to 2000 pending Child Workflows.
+
+:::
+
 ### SignalExternalWorkflowExecution
 
 This Command is triggered by a call to [Signal](/concepts/what-is-a-signal) another Workflow Execution.
 
 - Awaitable: Yes, a Workflow Execution can await on the action resulting from this Command.
 - Corresponding Event: [SignalExternalWorkflowExecutionInitiated](/references/events/#signalexternalworkflowexecutioninitiated)
 
+:::note
```

Comment:
Do we need to make these notes or is it is just information?

Reply:
It's repetitive; will remove.